src/scripts/stores/SurahsStore.js

```diff
@@ -88,9 +88,9 @@ SurahsStore.handlers = {
   },
 
   'NAVIGATE_START': function(payload, name) {
-    var currentRoute = this.dispatcher.getStore('RouteStore').getCurrentRoute();
+    let currentRoute = this.dispatcher.getStore('RouteStore').getCurrentRoute();
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).